configure.ac

```diff
@@ -111,7 +119,7 @@ AC_CONFIG_FILES( \
   plmpower/Makefile \
   etc/Makefile \
   scripts/Makefile \
-  scripts/powerman \
+  scripts/powerman.service \
```

Comment:
Do you need to distribute the generated `powerman.service` since it is built by `configure`?

Reply:
Great point, and _that's_ why in flux it's a `systemdsystemunit_DATA`.  I'll fix that.  Thanks for the review!